packages/react-instantsearch-hooks-web/src/ui/Stats.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+export type StatsTranslations = (
+  nbHits: number,
+  processingTimeMS: number,
+  nbSortedHits?: number,
+  areHitsSorted?: boolean
+) => string;
+
+export type StatsProps = React.ComponentProps<'div'> & {
+  nbHits: number;
+  processingTimeMS: number;
+  nbSortedHits?: number;
+  areHitsSorted?: boolean;
+  classNames?: Partial<StatsClassNames>;
+  translations: {
+    stats: StatsTranslations;
+  };
+};
+
+export type StatsClassNames = {
+  /**
+   * Class names to apply to the root element
+   */
+  root: string;
+};
+
+export function Stats({
+  classNames = {},
+  nbHits,
+  processingTimeMS,
+  nbSortedHits,
+  areHitsSorted,
+  translations,
+  ...props
+}: StatsProps) {
+  return (
+    <div
+      {...props}
+      className={cx('ais-Stats', classNames.root, props.className)}
+    >
+      <span className="ais-Stats-text">
+        {translations.stats(
+          nbHits,
+          processingTimeMS,
+          nbSortedHits,
+          areHitsSorted
+        )}
```

Comment:
I'm just doubting whether this may be a lot of parameters for one function. Maybe it should be put in an object, although we'd need to check what's consistent with the other translations.

Reply:
Can do - I only did it this way because it's the way `-dom` did it.